README.md

```diff
@@ -1,276 +1,129 @@
 # barnard59
 
-`barnard59` is a toolkit to automate extract, transform and load (ETL) like tasks. Its main focus is on creating [RDF & Linked Data](http://linked-data-training.zazuko.com/), a standardized graph data format by the W3C. barnard59 has a clear focus on creating RDF but is not limited to it.
+`barnard59` is a toolkit to automate extract, transform and load (ETL) tasks. Its main focus is on creating [Linked Data](http://linked-data-training.zazuko.com/). It allows you to generate RDF out of non-RDF data sources. In doing so, it follows the standard adopted in [Semantic Web](https://www.w3.org/standards/semanticweb/).
 
-It is built on many years of experience in creating RDF out of non-RDF data sources. Our goals & design principles are inspired by [the Unix philosophy](http://www.faqs.org/docs/artu/ch01s06.html):
+More specifically, `barnard59` is an engine to execute data pipelines.
 
-* *Describe the tasks in an abstracted way*. We use RDF to describe the so-called pipeline & steps, barnard59 is an engine to execute them.
-* *Enable executing code where necessary*. Sometimes plain code is faster & smarter than writing complex abstractions. You can execute in-line code in the pipeline in case you need expressivity.
-* *Automate everything*. barnard59 can be run within CI/CD toolkits like GitLab, run in a Docker container or simply within a node.js environment.
+## Pipeline
 
-barnard59 can be used for many different tasks, including, but not limited to
+A pipeline is a way to extract, transform, and load data (ETL). It allows you to consolidate data from various sources, and publish it as RDF.
+Using a pipeline you can:
+* download the source the data
+* validate it
+* transform it to linked data
+* publish it
 
-* Convert existing data to RDF. There is built-in support for CSV and JSON. Transformation is done in a streaming way when possible and thus can handle very large input data.
-* Shell & pipe support: Plug to existing command-line utilities you are comfortable with and work with their output.
-* Read from and write to SPARQL endpoints
 
-barnard59 can be easily extended by providing additional functionality as plugin. We also work on user interfaces, so you do not have to write your pipelines in Turtle/RDF.
+`barnard59` expects a pipeline to be defined as linked-data. The supported formats are:
+* text/turtle
+* application/ld+json
+* ...
```

Comment:
```suggestion
* `application/ld+json`
* `application/n-triples`
* `text/turtle`
* `application/rdf+xml`
```

From [@rdfjs/formats-common](https://github.com/rdfjs-base/formats-common/blob/master/index.js#L13-L19) without the Quads formats.